src/uct/ib/base/ib_iface.c

```diff
@@ -432,6 +433,7 @@ static ucs_status_t uct_ib_iface_create_cq(uct_ib_iface_t *iface, int cq_length,
     int env_var_added = 0;
     int ret;
 
+    pthread_mutex_lock(&lock);
```

Comment:
let's try to avoid setenv() at all - set CQE_SIZE only if HAVE_DECL_IBV_EXP_SETENV
otherwise revert to 64b (dfl) CQE